EWPP-284: Make sure theme can choose "core" and "standardised" ECL branding. (#586)
openeuropa/oe_theme: README.md

Note for developers: changing the component library will only load different CSS and JS assets, the actual HTML is the
same between the two libraries.

> Besides component libraries oe_theme support templates like "Core" and "Standardised". By default used "Standardised" templates.

Reviewer: Use this description instead:

```
Each component library can use one of the following ECL templates:

- "Standardized": standardised websites host thematic content owned by a specific DG/Agency. This is the default solution
  to host DG-specific content (policy) and is closely aligned with the core site.
- "Core": core websites host general information shared by different websites or departments and serve as hubs for
  onward navigation to further thematic content and/or specific services. For example, the main European Commission
  website (https://ec.europa.eu) uses ECL core templates.

ECL templates change the way users interact with the sites by restricting access to certain components, for example:
users can access to the main navigation menu only on sites using standardised ECL templates.

To learn more about ECL templates and component library families visit the (ECL website)[https://ec.europa.eu/component-library].
```

> ### Updating to 2.9.0+

Reviewer: Here add the following paragraph:

```
#### ECL templates

In 2.9.0 we introduced support for ECL templates (read above for more information). The OpenEuropa Theme will use the
"Standardised" template, visit the theme configuration page if you need to change that and use the "Core" ECL template instead.
```

**Note for developers**: changing the component library will only load different CSS and JS assets, the actual HTML is the
same between the two libraries.

Each component library can use one of the following ECL brandings:

- **Standardized**: standardised websites host thematic content owned by a specific DG/Agency. This is the default solution
  to host DG-specific content (policy) and is closely aligned with the core site.
- **Core**: core websites host general information shared by different websites or departments and serve as hubs for
  onward navigation to further thematic content and/or specific services. For example, the main European Commission
  website (https://ec.europa.eu) uses ECL core branding.

> ECL branding change the way users interact with the sites by restricting access to certain components, for example:

Reviewer: ```suggestion
ECL branding changes the way users interact with the sites by restricting access to certain components, for example:
```

users can access to the main navigation menu only on sites using standardised ECL branding.

> To learn more about ECL brandings and component library families visit the [ECL website](https://ec.europa.eu/component-library).

Reviewer: ```suggestion
To learn more about EC/EU families and ECL branding visit the [ECL website](https://ec.europa.eu/component-library).
```

- If your site does use a sub-them which displays the language switcher block, then you'll need to move it to the
  `site_header_secondary` region yourself

> #### ECL page header

Reviewer: Move this before  `### Upgrade to 2.9.0` with this:
```
 ### Upgrade to 2.10.0

 In 2.10.0 we dropped supporting the following elements in the ["Page header" pattern](./templates/patterns/page_header/page_header.ui_patterns.yml):
```

> In 2.9.0 we dropped supporting the following elements in the ["Page header" pattern](./templates/patterns/page_header/page_header.ui_patterns.yml):

Reviewer: ```suggestion
In 2.10.0 we dropped supporting the following elements in the ["Page header" pattern](./templates/patterns/page_header/page_header.ui_patterns.yml):
```

- `identity`: used to show site-identifying information (such as the site name).
- `infos`: used to show secondary meta information, below the page header introduction text.

As a result, if your `PageHeaderMetadata` plugins provide such data, it will no longer be displayed.

> #### ECL branding

Reviewer: Move this under the 2.10.0 section.

> In 2.9.0 we introduced support for ECL templates (read above for more information). The OpenEuropa Theme will use the

Reviewer: ```suggestion
In 2.10.0 we introduced support for ECL branding (read above for more information). The OpenEuropa Theme will use the
```

> "Standardised" template, visit the theme configuration page if you need to change that and use the "Core" ECL template instead.

Reviewer: ```suggestion
"Core" branding, visit the theme configuration page if you need to change that and use the "Standardised" branding instead.
```
